joomla/joomla-cms
# Fix the response from the AJAX ACL change to be correct

Related to #10755 
#### Summary of Changes

This change fixes the feedback given by the AJAX response when changing a permission.
#### Testing Instructions
1. Go to System -> Global configuration -> Permission
2. Click on Super User
3. Change the permission to Denied
4. The response is that the permission is Denied which is not true should be Allowed (Super User)
5. Click on Manager
6. Change Site Login to Denied
7. Click on Administrator
8. Change Site Login to Allowed
9. The response is that the permission is Allowed which is not true should be Conflict because the Denied of Manager overrules the Administrator setting
10. Apply the patch
11. Go to System -> Global configuration -> Permission
12. Click on Super User
13. Change the permission to Denied
14. The response is that the permission is Allowed (Super User). This is correct because the Super User setting overrules this.
15. Click on Manager
16. Change Site Login to Denied
17. Click on Administrator
18. Change Site Login to Allowed
19. The response is that the permission is in Conflict. This is correct because the Denied of Manager overrules the Administrator setting of Allowed

Calling @andrepereiradasilva and @infograf768 for special attention :)
